Update step 5 of pubsub-quickstart.md (#3966)
dapr/docs: daprdocs/content/en/getting-started/quickstarts/pubsub-quickstart.md

### Step 5: View the Pub/sub outputs

> The publisher pushes orders to the Dapr sidecar while the subscriber receives it.

Reviewer: ```suggestion
The publisher sends orders to the Dapr sidecar while the subscriber receives them.
```

Reviewer: Suggest a small wording change.  Thx
